frontend/src/components/search/SaveSearchPanel.tsx

```diff
@@ -0,0 +1,82 @@
+"use client";
+
+import { environment } from "src/constants/environments";
+import { useFeatureFlags } from "src/hooks/useFeatureFlags";
+import { useUser } from "src/services/auth/useUser";
+
+import { useTranslations } from "next-intl";
+import dynamic from "next/dynamic";
+import { usePathname, useSearchParams } from "next/navigation";
+import { useMemo } from "react";
+
+import { USWDSIcon } from "src/components/USWDSIcon";
+import { SaveSearchModal } from "./SaveSearchModal";
+import SearchQueryCopyButton from "./SearchQueryCopyButton";
+
+const TooltipWrapper = dynamic(() => import("src/components/TooltipWrapper"), {
```

Comment:
since the tooltip is being put in a different place in the logged in version, I moved the tooltip support stuff out to here